flaskr.py

```diff
@@ -80,4 +80,29 @@ def login():
 def logout():
 	session.pop('logged_in', None)
 	flash('You were logged out')
-	return redirect(url_for('show_entries'))
\ No newline at end of file
+	return redirect(url_for('show_entries'))
+
+"""From this point there are methods written by me 
+and not present in the turorial"""
+
+@app.route('/edit=<int:entry_id>', methods=['GET','POST'])
+def edit_entry(entry_id):
+	error=None
+	if request.method == 'POST':
+		db=get_db()
+		cur = db.execute('select title, text from entries where id = (?)', (entry_id,))
+		entries = cur.fetchall()
+		print(entries)
+		return render_template('edit_entry.html',id=entry_id,entries=entries)
+	return render_template('edit_entry.html',id=entry_id,entries=entries)
+
+@app.route('/edit=<int:entry_id>&edited=true', methods=['GET','POST'])
+def update_entry(entry_id):
```

Comment:
Thank you for the explanations! Very helpful indeed, tbh, I was wondering what's the difference between GET and POST and how do I use them, this is the exact portion of clear and understandable information that I needed.